app/src/main/res/layout/webview_fragment.xml

```diff
@@ -71,7 +71,7 @@
         android:textColor="@android:color/white"
         android:textStyle="bold"
         android:id="@+id/bannerCallActionNotice"
-        android:background="@color/colorPrimaryDark"
```

Comment:
Don't use a TextView as the background for the bottombar. Use something like a plain `View`. It has no use being a textview.

Reply:
That's my code. :speak_no_evil: 

Originally had plans to show some text there.

Reply:
@shahsurajk it's okay. You'll learn with time. 🙈

Reply:
@Jaseemakhtar Make the change, push and request PR.

Reply:
Okay.